src/lens/does-follow.js

```diff
@@ -0,0 +1,39 @@
+import { gql } from "@apollo/client/core";
+import { apolloClient } from "@/apollo-client";
+import { getAddress } from "@/ethers-service";
+
+const DOES_FOLLOW = `
+  query($request: DoesFollowRequest!) {
+    doesFollow(request: $request) { 
+			followerAddress
+    	profileId
+    	follows
+		}
+  }
+`;
+
+const doesFollowRequest = (followInfos) => {
+  return apolloClient.query({
+    query: gql(DOES_FOLLOW),
+    variables: {
+      request: {
+        followInfos,
+      },
+    },
+  });
+};
+
+const doesFollow = async (profileId) => {
```

Comment:
As you can see in the video, Apollo errors are horrible, because you have no clue about the problem.

I think we should get into the habit of checking the input we receive before making a grapql query, that way we can throw an error and make debugging easy.

```javascript
if (!profileId) {
    throw new Error('profileId is undefined');
}
```